enclave-manager/web/src/components/constants.ts

```diff
@@ -0,0 +1,5 @@
+export const KURTOSIS_PACKAGE_PARAMS_URL_ARG = "package-args"
+export const KURTOSIS_PACKAGE_ID_URL_ARG = "package-id"
+export const KURTOSIS_CREATE_ENCLAVE_URL_ARG = "create-enclave"
+
+export const KURTOSIS_PACKAGE_URL_BASE="/"
```

Comment:
I'd prefer to have constants defined near to where they are used (ie components/mo- arguably having a `constants` file in a `components/enclaves/modals`) directory is a bad smell anyway - it suggests we've started working quite a lot of application logic into the components (which tbf is what I've given you...)